packages/ui/src/domainModules/storage/utilities/index.js

```diff
@@ -0,0 +1,15 @@
+import { LEVEL_ROOM } from '../constants'
+
+export const extractNameWithFirstLevelParent = storageLocation => {
```

Comment:
think it's good to call the input `nestedStorageLocation` to make it clear it needs to be on the nested format